src/librepgp/stream-armor.cpp

```diff
@@ -386,7 +385,6 @@ armored_src_read(pgp_source_t *src, void *buf, size_t len, size_t *readres)
 
         if (memcmp(param->readcrc, crc_fin, 3)) {
```

Comment:
I'm concerned about comparing `crc_fin` with potentially uninitialized `readcrc` (if CRC wasn't read successfully).
Or, to be more precise, the warning "CRC mismatch" will not appear in an indeterministic manner (every once in 2^24 cases, but still...)
What do you think, @ni4

Reply:
`readcrc` is initialized to zeroes in https://github.com/rnpgp/rnp/blob/master/src/librepgp/stream-common.cpp#L358 by `calloc()`

Reply:
@rrrooommmaaa but at least "wrong crc line" will be printed anyway

Reply:
My point is: "CRC mismatch" should never be printed if "Wrong CRC line" was printed
or "CRC mismatch" should be printed every time when "Wrong CRC line" is printed.
In this implementation it won't be printed when "Wrong CRC line" was signaled and crc_fin == 0, right?
So looks like a glitch.

Reply:
Agree with @rrrooommmaaa - let's introduce some flag like `has_crc` and print different messages for different cases ("Warning: absent CRC", "Warning: malformed CRC", "Warning: CRC mismatch").
This may also help later in debugging.